tests/functional/tests/tests.yaml

```diff
@@ -1,5 +1,6 @@
 tests:
   - tests.functional.tests.model.COUModelTest
+  - tests.functional.tests.cou_smoke.COUSmokeTest
```

Comment:
not-blocking: instead of `cou_smoke` we could use `smoke` and instead of `COUSmokeTest` -> `SmokeTest` or better `CLITest`. I used `COUModelTest` only because the name of class is `COUModel`.